pkg/git/git.go

```diff
@@ -160,9 +149,7 @@ func plumbCommand(cmd *exec.Cmd, channel ssh.Channel, sidechannel io.Writer) *sy
 	}()
 
 	cmd.Stdout = channel
-	cmd.Stderr = channel.Stderr()
-
-	return &wg
```

Comment:
I take it the old waitgroup add calls were all removed at some intermediary point? At one point, the purpose of the wait group was to make sure that for large inputs, we could read in the background, but parallelize calls at a higher level. It looks like this may have eroded away over time, but I'm concerned that a big input could have some weird issues if this copy is still going on at the same time that sidechannel gets consumed.

More specifically, if this is 3/4 of the way through a write on the io.Copy operation, and you read to the end of the sidechannel file, will you get an EOF, or will it start blocking?